hr_period/views/hr_period_view.xml

```diff
@@ -43,7 +43,7 @@
         <field name="name">hr.period.tree</field>
         <field name="model">hr.period</field>
         <field name="arch" type="xml">
-          <tree colors="blue:state == 'open';gray:state == 'done'" string="Payroll Period">
+          <tree decoration-info="state == 'open'" decoration-muted="state == 'done'" name="Payroll Period">
```

Comment:
Why name instead of string?

Reply:
OCA guideline: The string attribute in list views is deprecated (no longer displayed) since version 8.0 [[source](https://github.com/OCA/odoo-community.org/blob/master/website/Contribution/CONTRIBUTING.rst#views)].

Reply:
That is so odd, there are very many similar usages in Odoo sources itself, while name is being used very rarely:
![image](https://user-images.githubusercontent.com/345102/62710771-8ffd0180-ba00-11e9-8126-3addab5aa5b1.png)

@pedrobaeza maybe you know why is that so?

Reply:
To be exact, from https://www.odoo.com/documentation/12.0/reference/views.html
![image](https://user-images.githubusercontent.com/345102/62712168-ff73f080-ba02-11e9-9688-3e9436f37916.png)
but no `name` mentioned as well.

Reply:
It seems that it's just ignored and can be simply removed 🤷‍♂

Reply:
So, `string` is not shown, neither does `name`. I don't like spaces in names as that is technical key, but that's not blocking :) Since I've approved the PR already, this one is just FYI :P